package.json

```diff
@@ -38,6 +38,7 @@
     "@types/eventemitter2": "^4.1.0",
     "@types/jest": "^26.0.23",
     "@types/puppeteer": "^5.4.3",
+    "@types/wicg-entries-api": "^2020.8.2",
```

Comment:
what's this for?

Reply:
this is for entries api used when handle drop file event.
spec: https://wicg.github.io/entries-api/